skyrim-platform/src/platform_se/skyrim_platform/TextsCollection.h

```diff
@@ -16,6 +17,14 @@ class TextsCollection
 
   void SetTextColor(int textId, std::array<double, 4> color);
 
+
+  void SetTextFont(int textId, std::string name);
+  void SetTextRotation(int textId, float rotation);
+  void SetTextSize(int textId, float size);
+  void SetTextEffect(int textId, int effect);
+  void SetTextDepth(int textId, float Depth);
+  void SetTextOrigin(int textId, std::array<double, 2> origin);
```

Comment:
It'd be better to also add blank lines between them for consistency

Reply:
fixed